css-text/Overview.bs

```diff
@@ -755,6 +755,9 @@ Languages and Typesetting</h4>
          the UA must either <a>hang</a> the <a>white space</a>
          or visually collapse their character advance widths
          such that they don't take up space in the line.
+         However, if 'overflow-wrap' is set to ''break-spaces'',
```

Comment:
When ''break-spaces'', spaces breaks, so it no longer be "at the end of a line". I think I understand your intention but it can confuse rather than help. Maybe you could add such note that this will not happen if ''break-spaces'' if you think having text here is important?